src/onnx/parse_generic_op.cpp

```diff
@@ -10,42 +10,23 @@ struct parse_generic_op : op_parser<parse_generic_op>
 {
     std::vector<op_desc> operators() const
     {
-        return {{"Abs", "abs"},
-                {"Acos", "acos"},
-                {"Acosh", "acosh"},
-                {"Asin", "asin"},
-                {"Asinh", "asinh"},
-                {"Atan", "atan"},
-                {"Atanh", "atanh"},
-                {"Ceil", "ceil"},
-                {"Concat", "concat"},
-                {"Cos", "cos"},
-                {"Cosh", "cosh"},
-                {"Elu", "elu"},
-                {"Erf", "erf"},
-                {"Exp", "exp"},
-                {"Flatten", "flatten"},
-                {"Floor", "floor"},
-                {"Gather", "gather"},
-                {"Identity", "identity"},
-                {"IsNaN", "isnan"},
-                {"LeakyRelu", "leaky_relu"},
-                {"Log", "log"},
-                {"LRN", "lrn"},
-                {"Neg", "neg"},
-                {"NonMaxSuppression", "nonmaxsuppression"},
-                {"Reciprocal", "recip"},
-                {"Relu", "relu"},
-                {"Round", "round"},
-                {"Scatter", "scatter"},
-                {"ScatterElements", "scatter"},
-                {"Sigmoid", "sigmoid"},
-                {"Sign", "sign"},
-                {"Sin", "sin"},
-                {"Sinh", "sinh"},
-                {"Sqrt", "sqrt"},
-                {"Tan", "tan"},
-                {"Tanh", "tanh"},
+        return {{"Abs", "abs"},          {"Acos", "acos"},
+                {"Acosh", "acosh"},      {"Asin", "asin"},
+                {"Asinh", "asinh"},      {"Atan", "atan"},
+                {"Atanh", "atanh"},      {"Ceil", "ceil"},
+                {"Concat", "concat"},    {"Cos", "cos"},
+                {"Cosh", "cosh"},        {"Elu", "elu"},
+                {"Erf", "erf"},          {"Exp", "exp"},
+                {"Flatten", "flatten"},  {"Floor", "floor"},
+                {"Gather", "gather"},    {"Identity", "identity"},
+                {"IsNaN", "isnan"},      {"LeakyRelu", "leaky_relu"},
+                {"Log", "log"},          {"LRN", "lrn"},
+                {"Neg", "neg"},          {"NonMaxSuppression", "nonmaxsuppression"},
+                {"Reciprocal", "recip"}, {"Relu", "relu"},
+                {"Round", "round"},      {"Sigmoid", "sigmoid"},
+                {"Sign", "sign"},        {"Sin", "sin"},
+                {"Sinh", "sinh"},        {"Sqrt", "sqrt"},
+                {"Tan", "tan"},          {"Tanh", "tanh"},
```

Comment:
I am not sure why clang-formatting change it to that. It should be a single list.

Reply:
I'll wrap this in clang-format off